src/window.rs

```diff
@@ -57,8 +57,11 @@ use serde::{Deserialize, Serialize};
 ///
 /// ## Platform-specific
 ///
-/// **Web:** The [`Window`], which is represented by a `HTMLElementCanvas`, can
-/// not be closed by dropping the [`Window`].
+/// - **Web:** The window is represented by a `HTMLElementCanvas`, and cannot
+///   currently be closed by dropping the [`Window`].
+/// - **Android:** Each window is spawned inside its own process, and as such
```

Comment:
Copy-pasting my reply in https://github.com/rust-windowing/winit/issues/696#issuecomment-1994577840:

> > on Android each window is run in its own process
> 
> Not at all, anything related to the same app (services, multiple `Activities` which are probably the closest synonym to `Windows`) should all run in the same process.
> 
> This is actually an open issue, where certain Android system interactions _can_ cause it (the Android system) to launch another `Activity` while one is already open whereas both `android-activity` and `winit` are assuming it is a singleton, and everything blows up.
> 
> It is however unlikely to have multiple windows open at the same time, bar tricks with multiple task stacks, split-window, popout windows and perhaps desktop mode.